test/Makefile

```diff
@@ -0,0 +1,59 @@
+tests = \
+	metadata \
+	pause \
+	play \
+	play-pause \
+	stop
+
+.PHONY: \
+	test \
+	$(tests) \
+	clean
+
+test: $(tests)
+
+$(tests):
+	set -e ; \
+	test=$@ ; \
+	exit_code="$$test.exit-code.log" ; \
+	stderr="$$test.stderr.log" ; \
+	rm -f "$$exit_code" "$$stderr" ; \
+	# This construct directs stderr to what it currently is \
+	# but also sends it to a file for checking afterwards. \
+	exec 3>&1 ; \
+	{ \
+	  ret=0 ; \
+	  "./$$test" 2>&1 >&3 || \
+	  ret=$$? ; \
+	  echo "$$ret" > "$$exit_code" ; \
+	} | \
+	tee "$$stderr" ; \
+	< "$$exit_code" read ret ; \
+	if [ -s "$$stderr" ] ; then \
+	  if [ -n "$$MPV_MPRIS_TEST_NO_STDERR" ] ; then
```

Comment:
This seems to be missing the trailing `\`.

Reply:
Fixed.